main.go

```diff
@@ -110,6 +118,21 @@ func main() {
 	}
 }
 
+// getUnrestrictedClient creates a client required for listing PVCs from all namespaces.
+func getUnrestrictedClient() client.Client {
+	var options client.Options
+
+	options.Scheme = runtime.NewScheme()
+	utilruntime.Must(clientgoscheme.AddToScheme(options.Scheme))
+	utilruntime.Must(v1.AddToScheme(options.Scheme))
```

Comment:
You are redefining the same add-to scheme that is already defined in `init()` (and you are even missing some). 
I will suggest refactoring `init()` to call a function `addAllSchemes` that adds everything to a given scheme then use it both there and here. 

For the init case it will be ``addAllSchemes(sceme)`` and here it will be `addAllSchemes(options.Scheme)`